lib/ssl/src/tls_connection.erl

```diff
@@ -618,13 +622,22 @@ init({call, From}, {start, Timeout},
     %% negotiated_version is also used by the TLS 1.3 state machine and is set after
     %% ServerHello is processed.
     RequestedVersion = tls_record:hello_version(Versions),
+    IsCertStatusRequested = maps:is_key(
```

Comment:
This variable is redundant as we already have this information in `ssl_options`. `ssl_options` is used to build the ClientHello message and we know that the ClientHello will have an ocsp_stapling extension when it is enabled in `ssl_options`.

Reply:
> This variable is redundant as we already have this information in `ssl_options`. `ssl_options` is used to build the ClientHello message and we know that the ClientHello will have an ocsp_stapling extension when it is enabled in `ssl_options`.

done